src/blocks/Share/Share.tsx

```diff
@@ -28,8 +29,19 @@ const icons: IconsProps = {
 
 const b = block('share-block');
 
-const Share = ({items, title}: ShareBlockProps) => {
+const Share = ({items, title, blockName = BlockType.ShareBlock}: ShareBlockProps) => {
     const {pathname, hostname} = useContext(LocationContext);
+    const defaultEvent = useMemo(
+        () => ({
```

Comment:
why do we hardcode this event is it ok?

Reply:
In my opinion it is okay. Looks like there is no reason to create `enum` with events names, because they (names) are used only once

Reply:
ok